### DISCUSSION TITLE
[Question] 连接自定义 proxy 出现 `UNABLE_TO_VERIFY_LEAF_SIGNATURE`

### DISCUSSION THREAD

**Discussion by WeihanLi** [lobehub/lobehub#977] [Self Hosting] [answered]
### 🧐 问题描述 | Proposed Solution

在使用自己部署的一个 openai proxy 出现 `UNABLE_TO_VERIFY_LEAF_SIGNATURE` 错误

```
{
  "endpoint": "",
  "error": {
    "cause": {
      "code": "UNABLE_TO_VERIFY_LEAF_SIGNATURE"
    }
  }
}
```

**Comment by arvinxx** [comment, marked as answer]
详见： [使用代理中转请求时（例如：Surge），报 `UNABLE_TO_VERIFY_LEAF_SIGNATURE` 错误](https://github.com/lobehub/lobe-chat/wiki/Common-Error.zh-CN#%E4%BD%BF%E7%94%A8%E4%BB%A3%E7%90%86%E4%B8%AD%E8%BD%AC%E8%AF%B7%E6%B1%82%E6%97%B6%E4%BE%8B%E5%A6%82surge%E6%8A%A5-unable_to_verify_leaf_signature-%E9%94%99%E8%AF%AF)
